diff --git a/src/ragger/utils/misc.py b/src/ragger/utils/misc.py
index e12943a..e100f50 100644
--- a/src/ragger/utils/misc.py
+++ b/src/ragger/utils/misc.py
@@ -101,7 +101,7 @@ def _is_root(path_to_check: Path) -> bool:
 
 def find_project_root_dir(origin: Path) -> Path:
     project_root_dir = origin
-    while not _is_root(project_root_dir) and not (project_root_dir / ".git").resolve().is_dir():
+    while not _is_root(project_root_dir) and not (project_root_dir / ".git").resolve().exists():
         project_root_dir = project_root_dir.parent
     if _is_root(project_root_dir):
         raise ValueError("Could not find project top directory")
